contrib/forms/assessment_intake/view.php

```diff
@@ -376,4 +412,18 @@
 </form>
 <?php
 formFooter();
+?> 
+
+    <textarea cols=100 rows=2 wrap=virtual name="referrals_or" ><?php echo stripslashes($obj{"referrals_or"});?></textarea><br><br>
+
+<?php /* From New */ ?>
+
+<br>
+<a href="javascript:top.restoreSession();document.my_form.submit();" class="link_submit">[Save]</a>
+<br>
+<a href="<?php echo $GLOBALS['form_exit_url']; ?>" class="link"
+ onclick="top.restoreSession()">[Don't Save Changes]</a>
+</form>
+<?php
+formFooter();
```

Comment:
This one is so weird because I don't have an explanation for how it happened. It's not like `cat ${file}` sometimes decides it wants to double up on some code. Interpreting control characters I understand, but this is a mystery to me, especially since it looks like the only one. Either way, a fix is on its way!